app/src/main/java/org/mozilla/fenix/compose/LightDarkPreview.kt

```diff
@@ -0,0 +1,8 @@
+package org.mozilla.fenix.compose
```

Comment:
this file needs the license

Reply:
ah I knew this file was looking a little empty. fixing